03-CSS/src/css/futur/futur-game-box.css

```diff
@@ -0,0 +1,73 @@
+body {
+    font-family: 'Keania One', cursive;
+    background-color:#269;
+    background-image: linear-gradient(white 2px, transparent 2px),
+    linear-gradient(90deg, white 2px, transparent 2px),
+    linear-gradient(rgba(255,255,255,.3) 1px, transparent 1px),
+    linear-gradient(90deg, rgba(255,255,255,.3) 1px, transparent 1px);
+    background-size: 100px 100px, 100px 100px, 20px 20px, 20px 20px;
+    background-position:-2px -2px, -2px -2px, -1px -1px, -1px -1px;
+}
+
+.gameBox {
+    border-radius: 4vh;
+
+    background: linear-gradient(to right bottom, var(--shadow), rgb(160, 76, 76));
+    box-shadow: -.5vh -.5vh 2vh -.2vh var(--dark-purple) inset,
+        .6vh .6vh .7vh .2vh var(--dark-purple),
+        0 0 1.5vh -.5vh var(--dark-purple);
+}
+
+.logo,
+.accent {
+    background-color: var(--green);
+    background: linear-gradient(to right, var(--green), var(--dark-purple));
```

Comment:
Ich steh total auf das giftgrüne leuchten dass du damit bekommst.
Ich würde bei anderen Elementen auch in die Kerbe schlagen und mit grünen Highlights Akzente setzen. 

Anhaltspunkte:
```
--purple-trans: hsla(0, 0%, 0%, 0.4)
--button-highlight: hsla(118, 100%, 63%, 0.3);
```

screen:
`box-shadow: 0 0 .8vh .2vh var(--dark-purple) inset, .3vh .9vh 2vh 0 hsla(118,100%,63%,.6), .6vh .6vh .9vh 0 var(--orange-trans), -.4vh -.7vh 1.4vh 0 var(--dark-purple-trans);`

gamebox:
`    background: linear-gradient(to right bottom, var(--shadow), rgb(160, 76, 76) 70%, hsla(118,100%,63%,1));`
Hier mit dem alpha Kanal des grüns zu spielen ist auch total trippy 😄 

body:
`    background-color: hsl(118, 70%, 19%);`